tests/test_project.py

```diff
@@ -262,9 +262,10 @@ def label2(job):
         assert len(b._label_functions) == 2
         assert len(c._label_functions) == 1
 
+    @pytest.mark.filterwarnings("ignore:always")
     def test_conditions_with_inheritance(self):
         """Tests the inheritance of pre/post conditions.
-
+        
```

Comment:
@kidrahahjo Notice that you added whitespace here. I will remove it for you so that this PR can move forward.